gocd/vagrant-publisher.gocd.yaml

```diff
@@ -0,0 +1,22 @@
+format_version: 3
+pipelines:
+  Vagrant.Publish:
```

Comment:
This only deals with openSUSE, maybe mention that? Most pipelines mention either openSUSE, Factory or Leap in their name

Reply:
I have renamed it to `vagrant-publisher.opensuse.gocd.yaml`.